packages/taiko-client/driver/chain_syncer/blob/syncer.go

```diff
@@ -315,7 +315,7 @@ func (s *Syncer) checkLastVerifiedBlockMismatch(ctx context.Context) (*rpc.Reorg
 		lastVerifiedBlockID = blockInfo.BlockId
 		lastVerifiedBlockHash = blockInfo.BlockHash
 	} else {
-		lastVerifiedBlockID = ts.BlockId
+		lastVerifiedBlockID = ts.BatchId
```

Comment:
I think it should be `ts.BlockId`?

Reply:
I'm checking and trying to make sure it.